clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -73,8 +73,8 @@ public void testSerialization() throws Exception {
                 createListOffsetResponse(),
                 MetadataRequest.allTopics(),
                 createMetadataRequest(Arrays.asList("topic1")),
-                createMetadataRequest(Arrays.asList("topic1")).getErrorResponse(1, new UnknownServerException()),
-                createMetadataResponse(1),
+                createMetadataRequest(Arrays.asList("topic1")).getErrorResponse(2, new UnknownServerException()),
+                createMetadataResponse(2),
```

Comment:
We should also test version 1. A few lines below we have:

```java
createMetadataResponse(0);
createMetadataRequest(Arrays.asList("topic1")).getErrorResponse(0, new UnknownServerException());
```

You can add a couple more lines there.